PWT/Cress/Tournament.yaml

```diff
@@ -0,0 +1,58 @@
+species: Crawdaunt
+setname: PWT-Cress Tournament
```

Comment:
Cress has 2 different rosters in PWT, and both of them have sufficient rep from gen 4 and before, why not add both? (That would mean renaming this one to Tournament 2)

Reply:
True, but 2 Pokemon from that 1st set have a move each that come from Gen V, making it impossible to have the required 3 Pokemon per set. I could attempt to replace one of the moves (either Bulldoze or Scald) but I was waiting more on if and when we can get Gen V+ moves in PBR before jumping the gun.

Reply:
ohhhhh i guess i didnt use my Brain

Reply:
No sweat, dude. Glad to help!